### ISSUE TITLE
Feature Request: an option to add vimrc file

### ISSUE THREAD

**Issue by coditva** [nparkanyi/gedit3-vim-mode#4]
An option to add a vimrc file would be very welcome. Love the plugin btw, works like a charm!

**Comment by philFernandez** [comment]
Yes, I would love to have the ability to remap some keys.. eg. `inoremap jj <Esc>`

Any idea how this could be implemented?

**Comment by JonLiuFYI** [comment]
Using a real vimrc file sounds cool, but there might be some unforeseen problems. This plugin doesn't mimic all vim behaviour, and not 100% faithfully, so some lines of the file would just be silently ignored or act incorrectly. A potential "softer" solution would be to add a settings window to the plugin, accessible from gedit's plugins list, for common ways people configure vim. (#15)

I made a separate issue for remapping keys, #14.
